examples/basic/gcp_vertex.yaml

```diff
@@ -31,20 +44,9 @@ spec:
   gcpCredentials:
     projectName: GCP_PROJECT_NAME  # Replace with your GCP project name
     region: GCP_REGION  # Replace with your GCP region
-    workloadIdentityFederationConfig:
```

Comment:
workload identity federation is more secured and recommended way but the setup is not trivial for the GCP vertex ai show case.